lib/Logging.js

```diff
@@ -124,7 +122,7 @@ class LogWrapper {
     _log(messageParts, type) {
         messageParts = this._formatParts(messageParts);
         if(this.logger == null) {
-            this.messages.push([{type, messageParts}]);
+            this.messages.push({type, messageParts});
```

Comment:
This looks unrelated

Reply:
*To the commit

Reply:
Yep, but it turned out this was doing the wrong thing and I felt it wasn't worth creating a PR just to fix a bug.

Reply:
At the least it should be part of a different commit, PR/branch history is less important.